ruby/Ice/config/client.rb

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+# Copyright (c) ZeroC, Inc.
+
+require 'etc'
+require 'Ice'
+
+# Load the code generated by the Slice compiler. The Slice module VisitorCenter maps to a Ruby module with the same
+# name.
+require_relative 'Greeter.rb'
+
+# Create an Ice communicator to initialize the Ice runtime. The communicator is destroyed at the end of the block
+# returned by Ice::initialize. This communicator is initialized using command line input and the "config.client"
+# file as its arguments.
+Ice::initialize(ARGV, "config.client") do |communicator|
+    # GreeterPrx is a class generated by the Slice compiler. We create a proxy from a communicator, a property found
+    # with the "Greeter.Proxy" specification with the address of the target object, and the "config.client" file that
+    # we referenced during initialization.
+    # If you run the server on a different computer, replace localhost in the string below with the server's hostname
```

Comment:
Need to update this comment, see C# version.

`replace localhost in the string...` that is from greeter but, here we use the config.